tests/notes.test.ts

```diff
@@ -0,0 +1,161 @@
+import { describe, test, expect } from '@jest/globals';
+import { Services } from '../api/services';
+
+const services = Services.getInstance();
+const authService = services.getAuthService();
+const notesService = services.getNotesService();
+const usersService = services.getUsersService();
+
+describe('SimpleApi/notes', () => {
+  test('should get all notes', async () => {
+    const notes = await notesService.getNotes();
+
+    expect(notes.status).toBe(200);
+    expect(notes.data).toBeDefined();
+  });
+
+  test('should get one note by id', async () => {
+    const notes = await notesService.getNotes();
```

Comment:
Can be moved to beforeAll func, and inside assign some obeject with the results of execution